repository_service_tuf_worker/repository.py

```diff
@@ -531,8 +553,8 @@ def add_targets(
 
             bin_targets[bins_name].append(db_target)
 
-        self._send_publish_targets_task(task_id)
-        self._update_task(bin_targets, update_state)
+        subtask = self._send_publish_targets_task(task_id)
```

Comment:
We are sure we would have the final state of the main task or it could be some intermediate state?

Reply:
If the `.apply_async()` fails, it raises an exception. So the main task will be`FAILURE`. The most common case is for some reason the access to the Broker fails.
So, if not, it will return the object.
The `subtask` Celery task object.
This object contains attribute implementations containing built-in "callbacks" to retrieve the task from celery.
For example, retrieve task status, results, etc.
You can use the return from `.apply_async()` or you can build the object using `celery.Task.AsyncResult(<taskid>)`